source/javascripts/controllers/awards_controller.coffee

```diff
@@ -12,6 +11,14 @@ EGA.controller "AwardsController", ($scope, $http, filterPipeline) ->
     countries: names: [], selected: []
     searchTerm: ''
 
+  $scope.categoryNames =
+    fire: 'Feuer'
+    water: 'Wasser'
+    air: 'Luft'
+    earth: 'Erde'
+    youth: 'Jugend'
+    other: 'Andere'
+
```

Comment:
I'd like to do that during our refactoring session, because otherwise I have to come up with some structure now that we're gonna remove again anyway during the refactoring.